LICENSE.md

```diff
@@ -0,0 +1,11 @@
+# Released under MIT License
+
+Copyright (c) 2013 Mark Otto.
+
+Copyright (c) 2017 Andrew Fong.
```

Comment:
🤔

Reply:
Good catch haha- who is the owner? Is it Google?

Reply:
Usually it's the repo owners, you could check out other projects to see what they do.